etc/ncp-config.d/samba.cfg

```diff
@@ -17,6 +17,11 @@
       "name": "Password",
       "value": "ownyourbits",
       "type": "password"
+    },
+    {
+      "id": "FILTER_BY_GROUP",
+      "name": "Filter by NC group:",
+      "value": ""
```

Comment:
maybe a suggest value `optional_group` and maybe some explanation in the description, but in general looks good to me.

Reply:
Sure. Is `OPTIONAL_GROUP_FILTER` okay?

Reply:
sounds good to me

Reply:
Ah, I misunderstood you earlier. I've updated the description and added a suggest value

Reply:
@nachoparker Let me know if my recent change implements what you had in mind (once you get to it - there's no hurry).